docs/website/adr/007-standardize-log-output.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: 7
+title: |
+  7. Standardize log output
+authors:
+- name: Mithril Team
+tags: [Accepted]
+date: 2024-04-07
+---
+
+## Status
+
+Accepted
+
+## Context
+
+* Logs from Mithril client CLI are redirected to stderr. Only the result of the command execution is in `stdout`. This makes it possible to exploit the result, see our [blog post](/dev-blog/2024/02/26/mithril-client-cli-output-breaking-change).
+* Mithril aggregator logs are always redirected to `stdout` which is not consistent with the Mithril client CLI log strategy.
```

Comment:
The context should explain more clearly that:
- ADR #2 is not completely relevant now
- We have migrated recently the logs in the client to `stderr`
- Aggregator mixes 2 types of CLI commands, some of which would benefit from the logs output to `stderr`
- Standardization would be welcome

Reply:
Done ✅